wondrous-app/pages/organization/[username]/members.tsx

```diff
@@ -49,7 +49,14 @@ function OrgMemberPage() {
       }}
     >
       <EntitySidebar>
-        <MemberRequests orgData={org} />
+        <MemberRequests
+          orgData={org}
+          userPermissionsContext={
```

Comment:
shouldnt pass this in, use context instead
 check out this example in  `components/Common/InviteLinkModal/OrgInviteLink.tsx`

```
  const orgBoard = useOrgBoard();
  const podBoard = usePodBoard();
  const board = orgBoard || podBoard;
  const userPermissionsContext = board?.userPermissionsContext;
  const permissions = parseUserPermissionContext({
    userPermissionsContext,
    orgId: board?.orgId,
    podId: board?.podId,
  });
```